consumer/pdata/common.go

```diff
@@ -471,6 +468,32 @@ func (am AttributeMap) ForEach(f func(k string, v AttributeValue)) {
 	}
 }
 
+// CopyTo copies all elements from the current map to the dest.
+func (am AttributeMap) CopyTo(dest AttributeMap) {
```

Comment:
We already have `AttributeValue.CopyFrom`. Would it make sense to also reverse this and name it `CopyFrom` for consistency of the API?

Reply:
I have it on my personal TODO :) I would go with CopyTo everywhere.

Reply:
Sounds good. CopyTo works fine, let's use it everywhere. 

Let's do this soon so that there is not much code accumulated that uses CopyFrom and breaks because of the change.

Reply:
Immediately after this PR :)